readthedocs/core/views/__init__.py

```diff
@@ -39,20 +39,8 @@ class HomepageView(TemplateView):
     def get_context_data(self, **kwargs):
         """Add latest builds and featured projects."""
         context = super(HomepageView, self).get_context_data(**kwargs)
-        latest = []
-        latest_builds = (
-            Build.objects
-            .filter(
-                project__privacy_level=constants.PUBLIC,
-                success=True,
-            )
-            .order_by('-date')
-        )[:100]  # yapf: disable
-        for build in latest_builds:
-            if (build.project not in latest and len(latest) < 10):
-                latest.append(build.project)
-        context['project_list'] = latest
         context['featured_list'] = Project.objects.filter(featured=True)
+        context['projects_count'] = Project.objects.count()
```

Comment:
I timed this against the prod database and it ran in ~20ms. If that's too expensive, we could consider caching it.

Reply:
Should be fine, but probably worth caching since that seems like a lot of time for no real reason. We can likely just use the Django view cache decorator at a minute or 5 minute interval, if we don't have anything much live updating!

Reply:
I don't know if we can cache the whole page especially for logged in users. They have the username/dashboard fragment at the top. Perhaps we could do something with the `cache` template tag.